### PR TITLE
Fixed addLines() being used instead of expected add()

### PR DESCRIPTION
**Pull request by MarcinOrlowski**
Fixes case reported in https://github.com/logisim-evolution/logisim-evolution/issues/929#issuecomment-907583946

### CODE REVIEW

File: src/main/java/com/cburch/logisim/fpga/hdlgenerator/LedArrayGenericHDLGeneratorFactory.java [hunk 1]
```diff
@@ -319,27 +319,27 @@ public static boolean requiresClock(char typeId) {
               + HDL.BracketOpen()
               + pin
               + HDL.BracketClose(),
-              rOn,
-              rOff,
-              pinName));
+                                      rOn,
+                                      rOff,
+                                      pinName));
```
Comment (BFH-ktt1) [inline src/main/java/com/cburch/logisim/fpga/hdlgenerator/LedArrayGenericHDLGeneratorFactory.java:324]: Just as side note, but again this space thingy....

File: src/main/java/com/cburch/logisim/std/memory/TFlipFlop.java [hunk 2]
```diff
@@ -73,7 +73,7 @@ public String ComponentName() {
     @Override
     public ArrayList<String> GetUpdateLogic() {
       return (new LineBuffer())
-          .add("%s s_next_state %s s_current_state_reg %sT;", HDL.assignPreamble(), HDL.assignOperator(), HDL.xorOperator())
+          .add("{{1}} s_next_state {{2}} s_current_state_reg {{3}}T;", HDL.assignPreamble(), HDL.assignOperator(), HDL.xorOperator())
```
Comment (BFH-ktt1) [inline src/main/java/com/cburch/logisim/std/memory/TFlipFlop.java:76]: Here a space is missing between {{3}} and T

**Review by BFH-ktt1 — CHANGES_REQUESTED** [review]

File: src/main/java/com/cburch/logisim/fpga/hdlgenerator/LedArrayGenericHDLGeneratorFactory.java (hunk 1 above)
Comment (MarcinOrlowski) [reply src/main/java/com/cburch/logisim/fpga/hdlgenerator/LedArrayGenericHDLGeneratorFactory.java:324]: Fixed.

File: src/main/java/com/cburch/logisim/std/memory/TFlipFlop.java (hunk 2 above)
Comment (MarcinOrlowski) [reply src/main/java/com/cburch/logisim/std/memory/TFlipFlop.java:76]: Fixed

**Review by BFH-ktt1 — APPROVED** [review]

**Pull request merged**